tools/tests/bundle-ignore-files.js

```diff
@@ -0,0 +1,22 @@
+const selftest = require('../tool-testing/selftest.js');
+const bundler = require('../isobuild/bundler.js');
+
+selftest.define("bundle-ignore-files", () => {
+  const patterns = bundler.ignoreFiles;
+  const matchingInputs = [
+    '.git/',
+    '.meteor/',
+    '.DS_Store',
+    '.aaabbb.swp',
+    'Icon.',
```

Comment:
Per https://github.com/meteor/meteor/pull/7835#discussion_r81148872

Could you change `Icon.` to `Icon\r`?